File: src/main/kotlin/com/stannis/parser/sln/VcxprojStructure.kt
package com.stannis.parser.sln

data class VcxprojStructure(
    var path: String,
    var listOfCppFiles: List<String>?,
    var listOfHeaderFiles: List<String>?,
    var listofIncludedModules: List<String>?,
    var listOfUnusedFiles: List<String>?
)

File: Interpreter/Interpreter/Models/serialize/VcxprojStructure.cs
using System.Collections.Generic;

namespace Interpreter.Models.serialize{
    public class VcxprojStructure
    {
        
        public string path { set; get; }

        public IList<string> listOfCppFiles = new List<string>();

        public IList<string> listOfHeaderFiles = new List<string>();

        public IList<string> listOfIncludedModules = new List<string>();

        public IList<string> listOfUnusedFiles = new List<string>();

    }
};